Source/v3Net/MeetupBot/MeetupBot.cs

```diff
@@ -246,7 +343,9 @@ private static async Task<List<TeamsChannelAccount>> GetOptedInUsers(TeamInstall
                 var isBot = string.IsNullOrEmpty(member.Surname);
                 optInInfo.TryGetValue(member.ObjectId, out UserOptInInfo optInStatus);
 
-                if ((optInStatus == null || optInStatus.OptedIn) && !isBot)
+                if (!isBot && 
+                    teamInfo.OptMode.Equals(TeamInstallInfo.OptInMode) &&
```

Comment:
Where do we set a team's Optin status?

Reply:
Looks like the default for OptMode is OptInMode. So, when we add the bot to a team, it will be optedIn by default. Didn't we want the default to be OptOut when the bot is added to an existing team?

Reply:
Currently you need to go to Cosmosdb and change the mode. We can change to optout, that is fine for me, this was a change made when the default was optin but CSE required optout.